src/vs/workbench/contrib/terminal/browser/terminalActions.ts

```diff
@@ -302,9 +301,8 @@ export function registerTerminalActions() {
 			});
 		}
 		async run(accessor: ServicesAccessor) {
-			const terminalService = accessor.get(ITerminalService);
-			terminalService.getActiveGroup()?.focusNextPane();
-			await terminalService.showPanel(true);
+			accessor.get(ITerminalGroupService).activeGroup?.focusNextPane();
```

Comment:
when would `activeGroup `be undefined? if there's no active group, is there a reason to show the panel?

Reply:
Looks like the existing behavior was to show it anyway which would create a new terminal so I guess we can just keep that